<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>magnifyWindow</class>
 <widget class="QWidget" name="magnifyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QCustomPlot" name="plotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_hide">
   <property name="icon">
    <iconset>
     <normaloff>D:/课程/毕设/项目/材料/icon/隐藏1.png</normaloff>D:/课程/毕设/项目/材料/icon/隐藏1.png</iconset>
   </property>
   <property name="text">
    <string>隐藏属性表</string>
   </property>
   <property name="toolTip">
    <string>隐藏属性表</string>
   </property>
  </action>
  <action name="action_unfold">
   <property name="icon">
    <iconset>
     <normaloff>D:/课程/毕设/项目/材料/icon/显示1.png</normaloff>D:/课程/毕设/项目/材料/icon/显示1.png</iconset>
   </property>
   <property name="text">
    <string>展开属性表</string>
   </property>
   <property name="toolTip">
    <string>展开属性表</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
